pkg/gateway/sig/sig.go

```diff
@@ -134,6 +134,11 @@ func IsAWSSignedRequest(req *http.Request) bool {
 	if strings.HasPrefix(v2Value, "AWS ") {
 		return true
 	}
+	// unauthenticated requests might still pass X-Amz-Date
+	dateHeader := headers.Get("X-Amz-Date")
+	if dateHeader != "" {
+		return true
+	}
```

Comment:
Why do we need this one? Is it related to a different bug?
In this PR we added another authentication based on our auth middleware - do we have an issue identify AWS S3 gateway request by the current headers?

Reply:
Good catch - I thought I could get away without setting the access key and secret - it won't pass that header. Removed.